static/src/javascripts/projects/admin/modules/abtests/abtest-report-item.js

```diff
@@ -84,6 +86,26 @@ define([
         var ophanUrl = 'https://dashboard.ophan.co.uk/graph/breakdown?ab=' + this.config.test.id;
         this.getElem('ophan').innerHTML = '<a href="' + ophanUrl + '"">graph</a>';
 
+        if (window.abConfigSwitches.abShowHypothesesDashboard) {
```

Comment:
Other (non-developer) users look at the dashboards (apparently, I haven't been able to find out exactly who yet), so it's better to keep it clean than confuse them with lots of useless statements I think.